UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/TensorApplier.cs

```diff
@@ -0,0 +1,143 @@
+﻿using UnityEngine.MachineLearning.InferenceEngine;
+using System.Collections.Generic;
+using UnityEngine.MachineLearning.InferenceEngine.Util;
+
+namespace MLAgents.InferenceBrain
+{
+    public interface TensorApplier
+    {
+        void Execute(Tensor tensor, Dictionary<Agent, AgentInfo> agentInfo);
+    }
+
+    /// <summary>
+    /// The Applier for the Continuous Action output tensor.
```

Comment:
This description is the equivalent of "Iterate over all elements" before a for-loop :). Not really adding much beside re-writing the method name. Should include info like, what is an Applier, define Tensor more concretely, etc.

Reply:
When the class name is so awesome you don't need a class comment. :)

Reply:
I will make a comment on the interface, but not sure one is needed here